.kokoro

```diff
@@ -40,6 +40,7 @@ fix_bazel_imports() {
   }
 
   stashed_dir=""
+  rewrite_tests "s/import MaterialComponents.Material(.+)_(.+)/import components_\1_\2/"
```

Comment:
Please merge https://github.com/material-components/material-components-ios/pull/3457/files first and then rebase this PR.

Reply:
Done.